uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Upgrader/Step/1021.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+trait CRM_HRLeaveAndAbsences_Upgrader_Step_1021 {
+  
+  /**
+   * Create new option group hrleaveandabsences_work_pattern_change_reason
+   * with values As per contract, Change in contractual hours and
+   * Change in contract type.
+   *
+   * @return bool
+   */
+  public function upgrade_1021() {
+    require_once 'CRM/Utils/Migrate/Import.php';
```

Comment:
I think this require_once is not really necessary. Can you please remove it and check if the upgrader will still work?

Reply:
It is not necessary. The upgrader still works after removing it.